docs/discovery.md

```diff
@@ -0,0 +1,54 @@
+
+## Dynamic Discovery
+The device service supports [dynamic discovery](https://docs.edgexfoundry.org/2.1/microservices/device/Ch-DeviceServices/#dynamic-provisioning).
+During dynamic discovery, the device service scans all connected USB devices and sends the discovered cameras to Core Metadata.
+The device name of the camera discovered by the device service is comprised of Card Name and Serial Number, and the characters colon, space and dot will be replaced with underscores as they are invalid characters for device names in EdgeX.
+Take the camera Logitech C270 as an example, it's Card Name is "C270 HD WEBCAM" and the Serial Number is "B1CF0E50" hence the device name - "C270_HD_WEBCAM-B1CF0E50".
+
+> NOTE: Card Name and Serial number are used by the device service to uniquely identify a camera. Some manufactures, however, may not support unique serial numbers for their cameras. Please check with your camera manufacturer.
+
+### Enable the Dynamic Discovery function
+Dynamic discovery is disabled by default to save computing resources.
```

Comment:
In the configuration.toml (line 129) it appears that Device.Discovery is enabled by default.